includes/classes/Product.php

```diff
@@ -22,7 +22,6 @@ class Product
     /** @deprecated use ->get('property') or ->getData()  */
     public array $fields;
 
-    /** @deprecated use !exists()  */
```

Comment:
I don't want to remove this deprecation, because I don't want people depending on `$EOF`.